signer/src/storage/postgres.rs

```diff
@@ -584,8 +584,8 @@ impl PgStore {
 
         // Given the utxo candidate above, this is our best guess of the
         // minimum UTXO height. It might be wrong, we'll find out shortly.
-        let min_block_height_candidate = utxo_candidate
-            .block_height
+        let min_block_height_candidate = BitcoinBlockHeight::try_from(utxo_candidate.block_height)
+            .map_err(Error::ConversionDatabaseInt)?
```

Comment:
You can change the type of `PgCandidateUtxo.block_height` to be a bitcoin block height. No need to make the change for this PR though.

Reply:
Oh wait, we aren't waiting on me, so I suppose there is no harm in making the change here, and it's preferable to do so now.

Reply:
[f5efcf6](https://github.com/stacks-sbtc/sbtc/pull/1448/commits/f5efcf6eaa58292ebcec1ccd753783908e362ca6)